SPARQLWrapper/sparql_dataframe.py

```diff
@@ -1,50 +1,74 @@
 """
 Query a SPARQL endpoint and return results as a Pandas dataframe.
 """
-from SPARQLWrapper import SPARQLWrapper2, SPARQLWrapper, CSV, SELECT
+import io
+from typing import TYPE_CHECKING, Any, Dict, List, Union
+
+import rdflib.term
+
+from SPARQLWrapper.SmartWrapper import Bindings, SPARQLWrapper2, Value
+from SPARQLWrapper.Wrapper import CSV, SELECT, SPARQLWrapper
+
+if TYPE_CHECKING:
+    import pandas as pd
```

Comment:
This is causing the CI pipeline to fail:

```
$ .venv/bin/python3 -m mypy --show-error-context --show-error-codes SPARQLWrapper
SPARQLWrapper/sparql_dataframe.py:13: error: Skipping analyzing "pandas": module is installed, but missing library stubs or py.typed marker  [import]
        import pandas as pd
    ^
SPARQLWrapper/sparql_dataframe.py:13: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 7 source files)
```

```suggestion
    import pandas as pd  # type: ignore[import]
```

Reply:
f467d40